pkg/net/net.go

```diff
@@ -66,3 +69,26 @@ func NthIPInNetwork(network *net.IPNet, n int) (net.IP, error) {
 
 	return nil, errors.New("network does not contain enough IPs")
 }
+
+// DNSNames returns a default set of machine names. It includes the hostname,
+// and FQDN. If the kernel domain name is not set, no names will be returned.
+func DNSNames() ([]string, error) {
```

Comment:
super nit, feel free to ignore

should we change to named return types here and clean up the variable declaration?

```
func DNSNames()  (dnsNames []string, err error) {
  var (
    domainName string
    hostname string
  )
 ...
```

Reply:
feel like it helps make the function a bit more concise.

Reply:
👍 works for me.